build.gradle

```diff
@@ -201,6 +201,7 @@ configure(javaAndScalaTestedProjects) {
 
         if (project.buildingWith('gssapi.enabled')) {
             systemProperties(
+                    'org.mongodb.test.gssapi.login.context.name': project.getProperty('gssapi.login.context.name'),
```

Comment:
I was just following the pattern for `gssapi.enbabled` by making this a project property.